src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,58 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  movie: {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  },
+}) => (
```

Comment:
better destructure props in the component before return!

Reply:
i cannot understand what do you mean

Reply:
Like this or what?
```
export const MovieCard: React.FC<Props> = ({ movie }) => {
   const {
    title,
    description,
    imgUrl,
    imdbUrl,
  } = movie

```